keps/sig-network/3866-nftables-proxy/README.md

```diff
@@ -1635,14 +1614,7 @@ provide more information.
 
 ###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
 
-TBD; we plan to add an e2e job to test switching from `iptables` mode
-to `nftables` mode in 1.31.
-
-<!--
-Describe manual testing that was done and the outcomes.
-Longer term, we may want to require automated upgrade/rollback tests, but we
-are missing a bunch of machinery and tooling and can't do that now.
--->
+Tested by hand.
```

Comment:
Testing by hand is fine, but can you please elaborate more how/what exactly you tested?

This is quite verbose, but nice example of test description: https://github.com/mimowo/enhancements/blob/9303678d20aef13af6fdb85a98128deb7603cd94/keps/sig-apps/3850-backoff-limits-per-index-for-indexed-jobs/README.md#were-upgrade-and-rollback-tested-was-the-upgrade-downgrade-upgrade-path-tested